docs/cms/input-types.adoc

```diff
@@ -91,22 +91,45 @@ image::Content-selector-default.png[An input box with a dropdown options below i
   </config>
 </input>
 ----
-<1> *allowContentType* is used to limit the content types that may be selected using the <<allowContentType Samples>>
+<1> *allowContentType* is used to limit the content types that may be selected using the <<allowContentType>>
 <2> *allowPath* is used to limit the path of the content that may be selected using the <<allowPath Samples>>. By default, Content Selector
 only displays content from the same site.
 <3> *treeMode* toggles between tree and flat modes. Default value is `false` (flat mode). +
 image:Content-selector-tree.png[Input box with a dropdown with a tree structure, 350]
 <4> *hideToggleIcon* hides the toggle icon when set to `false`. +
 image:Content-selector-no-icon.png[Input box without an icon, 500]
 
-=== allowContentType Samples
+[#allowContentType]
+=== Allow content type pattern
+There are two modes of matching content type patterns in XP: `LEGACY` to simulate pattern matching before image:../images/xp-770.svg[XP 7.7.0,opts=inline] and `MATCH` as current default mode. It can be configured in <<../deployment/config.adoc#admin, config>>.
 
```

Comment:
I think this is a little unclear in how it's phrased. Could we clarify it a bit? Something like:

```suggestion
XP uses one of two modes when matching content type patterns:

`LEGACY`:: this mode simulates the pattern matching functionality that was the default before XP 7.7.0. image:../images/xp-770.svg[XP 7.7.0,opts=inline] 
`MATCH`:: this mode is the current default mode and has a few notable changes from `LEGACY` mode.

The modes are explained in further detail below. You can configure which mode to use in  <<../deployment/config.adoc#admin, the admin config>>.
```

And this might just be an error on my side, but when running the Asciidoc browser extension, it couldn't find the xp-770 svg. But this isn't run through any sort of build system or anything, so I don't know how it'll work on the doc page. I'm assuming you've got it under control, but just wanted to flag it 🙋🏾